notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<ul><li>Can we change the requirements to Python 3.6 now that we have support? </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>yeah, we should do that on other nbs too</p>